pkgs/tools/inputmethods/fcitx5/fcitx5-anthy.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, cmake, extra-cmake-modules, pkg-config, fcitx5, anthy, gettext }:
+
+stdenv.mkDerivation rec {
```

Comment:
Make sure to add new packages to `pkgs/top-level/all-packages.nix` with a statement like `fcitx5-anthy = callPackage ../tools/inputmethods/fcitx5/fcitx5-anthy.nix { };` so they can be reachable as `fcitx5-anthy`